src/panel/NoResultMessage.jsx

```diff
@@ -16,6 +20,15 @@ const NoResultMessage = () => {
           'suggest'
         )}
       </p>
+      <p className="u-center">
+        <Button
+          href={helpCenterLink}
+          variant="tertiary"
+          onMouseDown={() => window.open(helpCenterLink)}
+        >
+          <IconExternalLink /> {_('Add a missing place on the map')}
```

Comment:
Should the text only be in english ?

Reply:
Source strings are in english, and when we surround them with `_(...)`, it allows to translate them on transifex, but that can only be done after the feature is merged on master